site/src/components/table.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Table
+description: Table element and it's modifiers.
+template: standard
+---
+
+# Table
+
+Table element helps organize content into rows and columns. `table` class adds basic styles to the tables fast and easily. Adding supported modifiers will get a custom and responsive look to the table.
```

Comment:
The tablet element ....
The `table` class ...

Reply:
updated